docs/source/tutorials/investigate_robustness_starting_values.ipynb

```diff
@@ -1,145 +0,0 @@
-{
```

Comment:
You seem to have deleted `investigate_robustness_starting_values.ipynb` but  did not add `optimization_multiple_starting_values.ipynb`, so there is currently nothing in tutorial.

Reply:
Thanks a lot for catching this. Fixed with next commit here in a second.